enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/HighLimit.java

```diff
@@ -42,6 +42,7 @@
  */
 public class HighLimit extends BaseRecordFormats
 {
+    public static final String NAME = "high_limit";
```

Comment:
Can we also add the same public `NAME` field to LowLimitV3_0.java and remove `COMMUNITY_KEY` from `InternalRecordFormatSelector` and start referencing this new field?